src/plugins/displayLayout/DisplayLayoutToolbar.js

```diff
@@ -0,0 +1,130 @@
+                                {
+                                    name: 'Label + Value',
+                                    value: 'all'
+                                },
+                                {
+                                    name: "Label only",
+                                    value: "label"
+                                },
+                                {
+                                    name: "Value only",
+                                    value: "value"
+                                }
+                            ]
+                        },
+                        {
+                            control: "separator"
+                        },
+                        {
+                            control: "select-menu",
+                            domainObject: domainObject,
+                            property: path + ".value",
+                            title: "Set value",
+                            options: metadata.values().map(value => {
+                                return {
+                                    name: value.name,
+                                    value: value.key
+                                }
+                            })
+                        },
+                        {
+                            control: "separator"
+                        },
+                        {
+                            control: "color-picker",
+                            domainObject: domainObject,
+                            property: path + ".fill",
+                            icon: "icon-paint-bucket",
+                            title: "Set fill color",
+                            type: "fill"
+                        },
+                        {
+                            control: "color-picker",
+                            domainObject: domainObject,
+                            property: path + ".stroke",
+                            icon: "icon-line-horz",
+                            title: "Set border color",
+                            type: "stroke"
+                        },
+                        {
+                            control: "color-picker",
+                            domainObject: domainObject,
+                            property: path + ".color",
+                            icon: "icon-T",
+                            mandatory: true,
+                            title: "Set text color",
+                            preventNone: true
+                        },
+                        {
+                            control: "separator"
+                        },
+                        {
+                            control: "select-menu",
+                            domainObject: domainObject,
+                            property: path + ".size",
+                            title: "Set text size",
+                            options: [9, 10, 11, 12, 13, 14, 15, 16, 20, 24, 30, 36, 48, 72, 96].map(function (size) {
```

Comment:
This is kinda long, perhaps a `CONST_VAR` would be favorable? Also the map from earlier didn't use `function(val)` signature on its cb, perhaps a consistency there?

Reply:
I will update to use the arrow function here for consistency. Generally it's a better practice to define it as a constant regardless of the length.